client/src/components/DataFiles/DataFilesModals/DataFilesUploadModalListing/DataFilesUploadModalListingTable.jsx

```diff
@@ -58,7 +59,9 @@ function DataFilesUploadModalListingTable({
         <tbody>
           {uploadedFiles.map((file, i) => (
             <tr key={file.id}>
-              <td style={{ verticalAlign: 'middle' }}>{file.data.name}</td>
+              <td style={{ verticalAlign: 'middle' }}>
+                {submission ? '[File Name Redacted]' : file.data.name}
```

Comment:
I don't think we should hide the filename during upload. If a single upload were to fail during multiple file uploads, it would be important for the user to know which upload failed. The fact that the client already knows the filename means we aren't really hiding anything not already known to the uploader.

Reply:
Good point. I was confused by that requirement as well. We should ask Craig to clarify